internal/server/iigointernal/judiciary.go

```diff
@@ -149,6 +233,49 @@ func (j *judiciary) clearHistoryCache() {
 
 // Helper functions //
 
+func runEvaluationRulesOnSanctions(localSanctionCache map[int][]roles.Sanction, reportedIslandResources map[shared.ClientID]shared.Resources) map[shared.ClientID]shared.Resources {
```

Comment:
since this is going to be tested later as part of the integration, for now at least could get a description of what the function does a comment. Just so we know its intended purpose. There's alot of moving parts which are hard to keep a track of

Reply:
Yeah sorry, thanks for the spot. Added a description.

Reply:
I realised with a little adjustment we can add a test to this and move the RulesInPlay dependency upstream. The function is now tested.